### PR TITLE
EJ2-Documentation(860838) - UG Documentation of Rows

### PR DESCRIPTION
**Pull request by Gayathri4135**
UG Documentation of Rows

### CODE REVIEW

File: ej2-javascript/code-snippet/grid/custom-row-cs1/index.ts [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
 
+let smallHeightButton: Button = new Button({ cssClass: 'e-small' }, '#small');
+let mediumHeightButton: Button = new Button({ cssClass: 'e-small' }, '#medium');
+let bigHeightButton: Button = new Button({ cssClass: 'e-small' }, '#big');
 
-
+(document.getElementById('changeHeight')as HTMLElement).onclick=function(args:ClickEventArgs)
+{
+ let heightRow={small: 20, medium: 40, big: 60};
+  grid.rowHeight = (heightRow)[args.target.id];
```
Comment (BalajiSGrid) [inline ej2-javascript/code-snippet/grid/custom-row-cs1/index.ts:27]: need to set the typing on target

File: ej2-javascript/code-snippet/grid/row-hover-cs1/index.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+});
+grid.appendTo('#Grid');
+
+let toggle: Switch = new Switch({
+    checked: true,
+    change: onSwitchChange,
+});
+toggle.appendTo('#switch');
+
+function onSwitchChange(args: ChangeEventArgs) {
```
Comment (BalajiSGrid) [inline ej2-javascript/code-snippet/grid/row-hover-cs1/index.ts:22]: need to optimize this function codes
use this  "grid.enableHover = !args.checked"

File: ej2-javascript/code-snippet/grid/row-hover-cs1/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+});
+grid.appendTo('#Grid');
+
+var toggle = new ej.buttons.Switch({
+  checked: true,
+  change: onSwitchChange,
+});
+toggle.appendTo('#switch');
+
+function onSwitchChange(args) {
```
Comment (BalajiSGrid) [inline ej2-javascript/code-snippet/grid/row-hover-cs1/index.js:18]: need to optimize this function codes
use this  "grid.enableHover = !args.checked"

File: ej2-javascript/code-snippet/grid/row-pinning-cs1/index.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  value: 2,
+  width: '100px'
+});
+numeric.appendTo('#frozenRows');
+
+let button: Button = new Button({ content: 'Update' });
+button.appendTo('#updateButton');
+
+(document.getElementById('updateButton') as HTMLElement).addEventListener('click', function() {
+  let inputValue = parseInt((document.getElementById('frozenRows')as HTMLElement).value);
```
Comment (BalajiSGrid) [inline ej2-javascript/code-snippet/grid/row-pinning-cs1/index.ts:43]: Get the value using NumericTextBox component's instance

File: ej2-javascript/code-snippet/grid/row-pinning-cs1/index.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  format:'n',
+  value: 2,
+  width: '100px'
+});
+numeric.appendTo('#frozenRows');
+
+var button = new ej.buttons.Button({ content: 'Update' });
+button.appendTo('#updateButton');
+
+document.getElementById('updateButton').addEventListener('click', function() {
```
Comment (BalajiSGrid) [inline ej2-javascript/code-snippet/grid/row-pinning-cs1/index.js:37]: Get the value using NumericTextBox component's instance

File: ej2-javascript/code-snippet/grid/row-pinning-cs1/index.ts (hunk 4 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/row-pinning-cs1/index.ts]: Used the numbertextbox instance in sample

File: ej2-javascript/code-snippet/grid/row-pinning-cs1/index.js (hunk 5 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/row-pinning-cs1/index.js:37]: Used the numbertextbox instance in sample

File: ej2-javascript/code-snippet/grid/row-hover-cs1/index.js (hunk 3 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/row-hover-cs1/index.js:18]: changed the switchchange function code "grid.enableHover = args.checked"

File: ej2-javascript/code-snippet/grid/row-hover-cs1/index.ts (hunk 2 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/row-hover-cs1/index.ts:22]: changed the switchchange function code "grid.enableHover = args.checked"

File: ej2-javascript/code-snippet/grid/custom-row-cs1/index.ts (hunk 1 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/custom-row-cs1/index.ts]: set an typing on target

**Comment by BalajiSGrid** [comment]
Technical review completed

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**